### PR TITLE
Add support for Plone 5.1.x

### PR DESCRIPTION
**Pull request by mbaechtold**
🚧  ~~The tests are passing but the widget is not working as advertised in Plone 5.1.x yet.~~ 🚧

### CODE REVIEW

**Comment by busykoala** [comment]
@mbaechtold, do you remember what was not working as advertised?

**Comment by mbaechtold** [comment]
Mhhh, this was almost a year ago, I don't remember 😂 I just booted it again and it seems to work:

ScreenRecording

**Comment by busykoala** [comment]
❗️We should not merge the branch. Even though the test are passing it does not seem to work yet.

File: ftw/keywordwidget/profiles/default/jsregistry.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <object name="portal_javascripts">
 
     <javascript
-        id="++resource++ftw.keywordwidget/js/keywordwidget.js"
+        id="++resource++ftw.keywordwidget/js/keywordwidget_4.x.js"
```
Comment (maethu) [inline ftw/keywordwidget/profiles/default/jsregistry.xml]: This needs a upgrade step, which removes the keywordwidget.js and add the other one. 

`<javascript id="++resource++ftw.keywordwidget/js/keywordwidget.js" remove="True" />`

Comment (busykoala) [reply ftw/keywordwidget/profiles/default/jsregistry.xml]: I guess I'll just leave the old name like it was...?

Comment (busykoala) [reply ftw/keywordwidget/profiles/default/jsregistry.xml]: I renamed the file to its original filename again. Therefore we should not need an upgrade step.

**Comment by busykoala** [comment]
@maethu, @mbaechtold, can we merge this now?

File: ftw/keywordwidget/profiles/uninstall_plone5/registry.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records prefix="plone.resources/keywordwidget_javascript"
+             interface='Products.CMFPlone.interfaces.IResourceRegistry'
```
Comment (mbaechtold) [inline ftw/keywordwidget/profiles/uninstall_plone5/registry.xml]: Could we uniformly use the same apostrophe in the entire file?

File: docs/HISTORY.txt [hunk 3]
```diff
@@ -5,7 +5,9 @@ Changelog
 1.5.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for Plone 5.1.x. [mbaechtold]
+- Fit newer vers. of select2 into keywordwidget resources to prevent conflict
```
Comment (mbaechtold) [inline docs/HISTORY.txt]: What do we gain by abbreviating "version" with "vers."?

File: ftw/keywordwidget/profiles/uninstall_plone5/registry.xml (hunk 2 above)
Comment (busykoala) [reply ftw/keywordwidget/profiles/uninstall_plone5/registry.xml]: 👍

File: docs/HISTORY.txt (hunk 3 above)
Comment (busykoala) [reply docs/HISTORY.txt]: 👍

**Pull request merged**